pkg/eval/builtin_fn_misc.go

```diff
@@ -160,7 +161,30 @@ import (
 // bar
 // ```
 
-// TODO(xiaq): Document esleep.
+//elvdoc:fn sleep
+//
+// ```elvish
+// sleep $duration
+// ```
+//
+// Sleep for the specified duration.  A duration string is a possibly signed
```

Comment:
Sleep -> Sleeps. I'd also prefer this to avoid using "sleep" (which is not very rigorous). Go's doc uses "pauses", which is good. Also document that this pauses *at least* the given duration, especially since you mentioned that the OS may have a minimal granularity.

Don't use double spaces after the period.

Reply:
Done